Fix favicon configuration bug

zlaazlaa opened the pull request:
<!--
Hello 👋 Thank you for submitting a pull request.

To help us merge your PR, make sure to follow the instructions below:

- Create or update the documentation. (Should be made against the `main` branch)
- Create or update the tests.
- Refer to the issue you are closing in the PR description - fix #issue
- Specify if the PR is in WIP (work in progress) state or ready to be merged

Please ensure you read through the Contributing Guide: 
https://github.com/strapi/documentation/blob/main/CONTRIBUTING.md
-->

### What does it do?

The configuration in the documentation for changing the favicon is not functioning properly. I have updated a method that works well.

### Why is it needed?

Mitigated a bug.

### Related issue(s)/PR(s)

Details about the bug can be found at https://github.com/strapi/strapi/issues/19238.

Commit:
Fix shortcuts note again

Commit:
Add information about the period before project deletion when suspended

kkukelka committed:
chore: fix relative paths on models page (#1968)

dev-shetty committed:
fix: broken nodejs releases link (#1967)

Commit:
Add more precisions about the emails sent before project cancellation

pwizla committed:
Update docusaurus/docs/dev-docs/admin-panel-customization.md

pwizla committed:
Update docusaurus/docs/dev-docs/admin-panel-customization.md

pwizla committed:
Update docusaurus/docs/dev-docs/admin-panel-customization.md

pwizla committed:
Update docusaurus/docs/dev-docs/admin-panel-customization.md

pwizla committed:
Update docusaurus/docs/dev-docs/cli.md

pwizla committed:
Update docusaurus/docs/dev-docs/cli.md

pwizla committed:
Update docusaurus/docs/dev-docs/cli.md

seanburlington committed:
Update authentication.md (#1970)

let the developer know they need to install formik to run this example

pwizla committed:
Update authentication.md

Commit:
Merge pull request #1924 from strapi/user/moving-blocks

Blocks improvements: moving blocks & expand field

Commit:
Bump version to 4.17.0

pwizla committed:
Add default column to options tables

Commit:
Add exhaustive list of markdown shortcuts in Blocks documentation

pwizla committed:
Updates to the Releases feature for the beta release (#1971)

* Mention grouping

* Update badge

* Update docusaurus/docs/user-docs/releases/introduction.md

* chore: update screenshot and add callout for beta version (#1972)

* Move beta callout to grouping section

* Update docusaurus/docs/user-docs/releases/managing-a-release.md

Co-authored-by: markkaylor <mark.kaylor@strapi.io>

* Update docusaurus/docs/user-docs/releases/managing-a-release.md

---------

Co-authored-by: markkaylor <mark.kaylor@strapi.io>

pwizla committed:
Update docs for autogenerate settings for JS projects

pwizla committed:
Revert "Update docs for autogenerate settings for JS projects"

This reverts commit 0a446e07a857aeabb5c9ab03a9e7188241766291.

pwizla committed:
Revert unrelated change

pwizla committed:
Mention API prefix config. in REST API intro.

Abdullahxz committed:
fixed sso okta caution block placement (#1976)

thomasbnt committed:
Err code with `./extensions` (#1977)

Because I tested this example to replace the favicon, and I've got an error.

```bash
yarn run v1.22.19
$ strapi build
⠋ Building build context
[INFO] Including the following ENV variables as part of the JS bundle:
    - ADMIN_PATH
    - STRAPI_ADMIN_BACKEND_URL
    - STRAPI_TELEMETRY_DISABLED
✔ Building build context (73ms)
⠋ Building admin panel
⠴ Building admin panel[ERROR] Module not found: Error: Can't resolve './extensions/favicon.png' in '/var/www/strapi/src/admin'
✖ Building admin panel
[ERROR]  There seems to be an unexpected error, try again with --debug for more information

error Command failed with exit code 1.
```

pwizla committed:
Mention other front-end renderers available on npm

pwizla committed:
Tweak link from previous commit

pwizla committed:
Slightly tweak wording from prev. commit

pwizla committed:
Update configuring-fields-content-type.md

pwizla committed:
Remove badges and mentions of future flags

pwizla committed:
Remove mention of beta for grouping

pwizla committed:
Fix headings

pwizla committed:
Fix icon

pwizla committed:
Mention validation errors

pwizla committed:
Additional fixes

pwizla committed:
Temporarily hide features config. docs

pwizla committed:
Update with status column and validation errors

Commit:
Strapi Cloud: Node version feature (#1978)

* Document Node version feature in deployment guide

* Update advanced settings screenshots

Commit:
Reviewed & complete version of Strapi Cloud documentation (#1961)

* Rework TOC titles

* Separate account/sessing into 2 pages

* Start working the Projects section

* Rework and complete Usage and Billing page

* Rework Projects overview page

* Fix links in Usage and Billing

* Separate deployment and deploy history in 2 pages and rework content in both

* Move deployment pages to a new Deployments section in the TOC

* Review Overview and Settings pages

* Review or write content for deploys docs and runtime logs

* Delete custom plan from Billing & Usage page

* Mention GitLab where it was forgotten

* Improve Notifications page

* Fix/improve all projects management pages

* Fix/improve all deployments pages

* Fix/improve Account settings page

* Update all screenshots

* Fix broken links

* Fix header link

* Remove useless link in Project settings

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>

* Update one more screenshot + Delete useless comments

* Add icons

* Delete More.svg in static

* Delete more.svg again

* Add More.svg

* Fix icon name in releases

* Add Node version feature in Project settings + Update screenshot

* Add forgotten word in deploy logs

---------

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>
Co-authored-by: Pierre Wizla <pwizla+github@gmail.com>

pwizla committed:
Update package.json

Commit:
Strapi Cloud: mandatory billing details (#1981)

* Update account billing page

* Update screenshot

pwizla committed:
Apply commit from #1982

pwizla committed:
Apply fix from #1972 to email config. too

zlaazlaa committed:
Update admin-panel-customization.md

strapi-cla commented:
[![CLA assistant check](https://cla.strapi.io/pull/badge/not_signed)](https://cla.strapi.io/strapi/documentation?pullRequest=1983) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.strapi.io/strapi/documentation?pullRequest=1983) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.strapi.io/check/strapi/documentation?pullRequest=1983) it.</sub>

pwizla commented:
Hi @zlaazlaa and thanks for the update.
It looks good to me. I'll hold off merging this PR until the corresponding strapi/strapi PR is merged.

zlaazlaa commented:
> Hi @zlaazlaa and thanks for the update. It looks good to me. I'll hold off merging this PR until the corresponding strapi/strapi PR is merged.

Thanks a lot❤️

pwizla committed:
Mention that only collection type entries can be added to releases

pwizla committed:
Update headings to avoid confusion re. 'single'

pwizla committed:
Fix sizeLimit TS code example for upload plugin

pwizla committed:
Fix capitalization

pwizla committed:
Merge branch 'main' of github.com:strapi/documentation

pwizla committed:
Merge branch 'main' into cloud/backups-UI

Commit:
Add note about maintainers vs project owners for restore backup feature

Commit:
Merge pull request #1969 from strapi/cloud/expansion-suspension-period

Strapi Cloud: extension of period before project deletion when suspended

Commit:
Merge pull request #1928 from strapi/cloud/backups-UI

Strapi Cloud: restore backup via UI

Commit:
Merge pull request #1973 from strapi/user/blocks-shortcuts

Add exhaustive list of Markdown shortcuts in Blocks documentation

Commit:
Merge pull request #1980 from strapi/user/releases-stable-updates

Updates for the stable version of Releases

Commit:
Merge pull request #1963 from strapi/feat/add-vite

Add vite bundler documentation to admin panel customisation

Commit:
Merge pull request #1975 from strapi/dev/autogenerate-updates

Update docs for autogenerate settings for JS projects

Commit:
Update package.json

pwizla committed:
Merge branch 'user/releases-stable-updates'

pwizla committed:
Merge branch 'main' of github.com:strapi/documentation

pwizla committed:
Fix more.icon

pwizla committed:
Try again to fix "more" icon 🤞

pwizla committed:
Fix Tab → TabItem component name

pwizla committed:
Revert moremore.svg icon naming to more.svg

pwizla committed:
Introduce Strapi Cloud in the Quick Start Guide (#1979)

* Add first draft of QSG updates for Strapi Cloud

* Add missing intro. sentence

* Update screenshot

* Fix caution callout rendering in data transfer

* Rewrite and reformat the QSG entirely in a logical and easier way

* Fix yarn/NPM casing

* Add light/dark mode for first screenshot

* Add light/dark mode for first GIF

* Add git to prerequisites

* Add git init step

* Link to official GH docs

* Update wording for free trial Cloud plan

* Update Rich Text wording

* Clean up comments

* Update first screenshot's light mode

* Update first GIF's light mode

* Remove unnecessary comma

* Remove mentions of "plugin" for CTB and CM

* Add or update all missing screenshots and GIF for both light & dark mode

* Mention image zoom bug for details blocks in config. file

* Remove unwanted GIF

* Simplify GH toggle title

* Add missing word

* Move sentence up

* Replace many-to-many icon

* Simplify Settings (icon) naming

* Fix style for links in details blocks in dark mode

stromseng committed:
update heroku cli ubuntu install command (#1986)

nikita-fuchs committed:
mention local file upload path (#1990)

To save others time researching this information I just added where the standard upload plugin is writing the upload files.

pwizla committed:
Warn against using locale for custom attribute name in Dev Docs

pwizla committed:
Warn against using locale for a custom attribute name in User Guide

pwizla committed:
Update wording from prev. commit

In the User Guide we use "fields" to represent "attributes"

pwizla committed:
Add link to Paul's blog post about TS in front-end

pwizla committed:
Bump version to 4.19.1

pwizla committed:
Remove unneeded callout for Audit Logs being EE

MbonuJennifer committed:
Adjusted Typescript Typo fix (#1996)

Co-authored-by: MbonuJennifer <jennifer.mbonu@ietech.com.ng>

Commit:
Added new dependencies (#2001)

pwizla committed:
Remove outdated mention about single types

pwizla committed:
Remove Content Source Map / Vercel Visual Editing documentation (#2003)

pwizla committed:
Fix rendering issue in CLI reference page

Commit:
Merge pull request #2002 from strapi/user/releases-adding-single-types

Update Releases documentation for next release

Commit:
Update package.json

pwizla committed:
Replace images with stripped EXIF data (#2005)

pwizla committed:
Mention formatter disabling in contribution guide

pwizla committed:
Update wording from last commit

Commit:
Update usage & billing page

Commit:
Finish updating Usage & Billing page

Commit:
Fix info about overages rates

Commit:
Mention Developer plan in Project creation

Commit:
Mention Developer plan for Backups

Commit:
Improve links

Commit:
Update project settings to mention upgrade or downgrade

Commit:
Improve wording

Commit:
Add information regarding downgrading in Account billing details

Commit:
Update screenshots to show Dev plan

Commit:
Add updated badges in TOC

Commit:
Fix mistake about Dev plan and Backups

Commit:
Add information about backups when downgrading

Commit:
Fix broken link

Commit:
Fix overages rate for asset storage

pwizla committed:
Add a small callout for good practices re. controllers vs. middlewares (#2017)

eskres committed:
Update admin-redirect.md (#2018)

src folder is in the parent directory rather than the config directory

Commit:
Merge pull request #2013 from strapi/cloud/dev-tier

Strapi Cloud: Developer plan

Commit:
Update package.json

eskres committed:
Update users-permissions.md (#2019)

Fixed url which did not match the route example at https://github.com/strapi/strapi-examples/blob/master/examples/login-react/src/App.js

joshuaellis committed:
feat: add plugin cli docs page (#1965)

* feat: add plugin cli docs page

* chore: edit create-a-plugin page

* Fix unwanted line indentation changes

* Split content into a guide and a reference

* Add guide to plugins development intro.

* Add Plugin CLI docs to TOC

* Fix capitalization

* Fix SEO for reference page

---------

Co-authored-by: Pierre Wizla <pwizla+github@gmail.com>

pwizla committed:
Mention experimental Plugin CLI in plugin creation page

Commit:
Added .dockerignore to documentation as this is essential when using compose and or building dockerfiles (#2026)

Commit:
Strapi Cloud: reconnect a repo (#2023)

* First draft to add reconnect a repo in project settings documentation

* Finish documenting reconnect a repo

* Update screenshot

* Add caution callout

ramadanomar committed:
chore: node 20 installation (#2004)

* chore: node 20 installation

* misc: changed to v20 everywhere

* remove autoformatting

pwizla committed:
Bump version to 4.20.2

pwizla committed:
Update sidebar with badges

bootleq committed:
Fix wrong default value of develop `--open` option (#2029)

- The default is `true`.
- Move it from "Description" to dedicated "Default" column.

pwizla committed:
Add new callout to Dev Docs intro

pwizla committed:
Add callout to User Guide intro

pwizla committed:
Add "where is my content callout?" to CLI install. docs

xylocone committed:
fix: typo (#2031)

pwizla committed:
Fix typo in DigitalOcean guide

pwizla committed:
Improve the Plugins configuration section to mention Upload/Media Library configuration (#2032)

* Mention config/plugins is used for Upload and GraphQL too

* Improve config/plugins introduction

* Improve GraphQL config. section

* Add Upload config. section

* Improve examples

* Update sidebar

pwizla committed:
Fix typo in Dev Docs intro.

pwizla committed:
Update CONTRIBUTING.md

MbonuJennifer committed:
Created new Dev badge and Updated documentation with appropriate badges (#2020)

* Created CloudDevBadge

* Added badges

* Added Strapi Cloud Team Badge to the following pages - Audit Logs and Review Workflows. Removed the badges from the account billing page

* Update docusaurus/docs/cloud/projects/settings.md

* Added Strapi Cloud Team Badge to the Review Workflow page

* Changed the cloud dev badge text color and background color

* Changed Strapi Dev to Green and ndented the images

* Fixed the italics symbol

* Rearranged the badges in SC order

---------

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>

pwizla committed:
Update Releases documentation with Scheduling (#2025)

* Remove completed TODO

* Mention scheduling in "Creating a release" page

* Add TODOs for screenshot updates

* Add future flags docs for scheduling

* Update introduction to Releases

* Update Creating a release with scheduling

* Update Managing a release with scheduling

* Mention release scheduling in CM Edit View

* Fix Releases UI icon

* Add webhook documentation

* Add Yannis' suggestion

* Add caution about needing D&P for Releases to work

Commit:
Update package.json

derrickmehaffy committed:
Add docs for disabling the growth experiments

derrickmehaffy committed:
Merge pull request #2035 from strapi/dev/adminFlags

Add docs for disabling the growth experiments

pwizla committed:
Mention Upload plugin API in REST API

pwizla committed:
Add missing closing parenthesis to last commit

MbonuJennifer committed:
Created an Introduction page for the User Guide and the General Settings Section (#2024)

* Revised the user guide introduction, separating other concepts from introduction.

* Fixed the typos and broken links on the User Guide Welcome page

* Fixed the typos and broken links on the User Guide - Setting up the admin panel page and User Guide fundamentals

* Created Introduction for User Docs general Settings

* Fixed typos and broken links in General Settings  Introduction

* Fixed more typos

* Fixed more typos and added General Settings Introduction Link

* Updated docusaurus/docs/user-docs/intro.md - Included a period in the General Settings card description.

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>

* Update docusaurus/docs/user-docs/getting-started/user-guide-fundamentals.md to include that v4 would be maintained up to 12 months after v5 has been released.

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>

* Update docusaurus/docs/user-docs/intro.md - the description in the Media Library Card

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>

* Updated the Getting started introduction

* Updated the Getting started introduction

* Updated Sidebars.js

* Updated sidebar.js file, user guide fundamentals and user guide introduction

* Updated General Settings Images

* Updated General Settings Introduction

* Updated general settings introduction

* Change transfer tokens images to include new token

* Updated general settings introduction

* Updated the license and pricing plans section in the user guide fundamental file to include badges for Cloud Dev, Cloud Pro, and Cloud Team features

* Removed whitespace in introduction section

* Fix typo

---------

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>

Commit:
Replace screenshots with new signup page (#2033)

pwizla committed:
Convert alpha badge to beta (#2039)

pwizla committed:
Bump version to 4.20.4

pwizla committed:
Fix previous commit 😅

Zetjen committed:
🔌 Upload plugin backend example (#2043)

* 🔌 Upload plugin backend example

* Removing wrong char

* Update docusaurus/docs/dev-docs/plugins/upload.md

---------

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>

nikita-fuchs committed:
Clarify generate types command usage for TypeScript (#2042)

* Update models.md

Add example command for more context and clarification. the previous wording was confusing, had to google and find this https://docs.strapi.io/dev-docs/typescript#generate-typings-for-project-schemas to make sense of the previous wording.

* Add yarn equivalent

---------

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>

Commit:
Strapi Cloud: new introduction & fundamentals page (#2040)

* Add docs intro callout in Cloud intro page

* Reorganise introduction + Add cards to main pages

* Clarify a bit what is Strapi Cloud

* Create Strapi Cloud fundamentals page for important concepts

Commit:
Strapi Cloud: removing one-time password in "Managing subscriptions" (#2046)

* Remove mention of the one-time password in subscription management procedure

* Rework document to simplify procedure

pwizla committed:
Bump version to 4.20.5

pwizla committed:
Clarify `populate` usage (#2034)

* WIP: Add beginning of "populate" guide and REST API populate ref. rewrite

* Add missing titles to 1vs.2 level(s) deep comparison

* WIP

* Another WIP

* Yet another WIP: Update populate guide + add guides section and more

* Make guides section not collapsable in sidebar

* Rework detailed population strategy

* Fix typo in a card of the Guides intro.

* Improve existing content of the Understanding populate guide

* Update sidebars

* Finish Understanding populate guide 🥳

* Simplify populate docs in REST API reference (+ table)

* Fix typo

* Fix another typo

* Update level(s) → levels

* tweak populate creator example to use route middlewares instead

* Fix layout and add deeper nested headings for easier navigation

---------

Co-authored-by: Derrick Mehaffy <derrickmehaffy@gmail.com>

pwizla committed:
Fix messing up with TOC for REST

pwizla committed:
Mention populate deep and fix typo

Commit:
Cloud documentation update & fix (#2059)

* Update Usage & Billing page to mention backups

* Delete Dev badge for backups

pwizla committed:
Improve intro. sentence for populate guide

pwizla committed:
Slightly update REST API intro.

MbonuJennifer committed:
Add callout to lifecycle hooks to cross-link to error handling docs (#2058)

* Updateded the models.md file

* Fixed grammar

pwizla committed:
Add blog link to populate article by Paul

derrickmehaffy committed:
Add docs for keycloak users-permissions provider (#2048)

* feat: Add docs for keycloak users-permissions provider

* Update docusaurus/docs/dev-docs/plugins/users-permissions.md

* Update docusaurus/docs/dev-docs/plugins/users-permissions.md

* Update docusaurus/docs/dev-docs/plugins/users-permissions.md

---------

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>

pwizla committed:
Updates for stable release of Releases Scheduling (#2061)

* Updates for stable

* Update new release screenshot

Wrong size

* Hide experimental flags

* Fix heading link

No badges anymore

pwizla committed:
Bump version to 4.21.0

pwizla committed:
Add banner for Strapi 5

pwizla committed:
Update QSG command to beta

pwizla committed:
Revert "Update QSG command to beta"

This reverts commit 86bdf194cd71a07e1a58ad466727e90d0377a190.

pwizla committed:
Update banner link to v5 what's new page

raphaelluethy committed:
fix: fix a version issue that happens on the newer builds of alpine-lunix (#2063)

Originally, I stumbled on an error with `../src/pipeline.cc` where it complained about an invalid struct type.

After some searching I found the issue:
The current image used as base in the Dockerfile example links to an alpine-linux version **later** than 3.18, which has made changes in `libvips`, as discussed in this PR: [Link](https://github.com/nodejs/docker-node/issues/2009)

The PR also mentions, that fixing the build to version 3.18 resolve the issue, which I also confirmed in my local setup when trying to deploy strapi through docker compose.

pwizla committed:
Update .sclt-changelog.json

nc1z committed:
Fix table overflow behavior on mobile (#2062)

* fix: used lighter color for dark mode code element

* fix: table overflow on mobile

kibwashere committed:
add a small note about the overages project deletion check (#2064)

pwizla committed:
Include blog post link to Unit Testing article

pwizla committed:
Add new guides section to REST API (#2065)

pwizla committed:
Bump version to 4.21.1

pwizla committed:
Customize Cloud Backup anchor id

saitheninja committed:
Fix: Typo. Balance some brackets. (#2068)

abhisharkjangir committed:
Adding bash command to list strapi controllers (#2069)

pwizla committed:
Uncomment instructions for list view editing

pwizla committed:
Revert "Adding bash command to list strapi controllers (#2069)"

This reverts commit 205820c49da972b81f498557930c9a63657ada37.

pwizla committed:
Revert "Revert "Adding bash command to list strapi controllers (#2069)""

This reverts commit 03a1ea788d13bb1a50de56860f062cdb6ba7671a.

pwizla committed:
Document bulk add to releases - CM list view (#2070)

pwizla committed:
Bump version to 4.22.0

pwizla committed:
Remove completed TODO

pwizla committed:
Mention where images are stored

pwizla committed:
Fix wrong link in admin panel config.

SpeedAD committed:
chore(): small chore to update upload path on ~/documentation/docusaurus/docs/dev-docs/deployment/snippets-proxy/admin-redirect.md (#2072)

pwizla committed:
Fix #2052 (#2075)

noahismith committed:
Fix missing internal documentation link (#2077)

pwizla committed:
Bump version to 4.23.0

MbonuJennifer committed:
Improve wording for the “Improve this page” link (#2079)

* Changed improve this page wording

* Fixed error - removed comments

pwizla committed:
Remove emoji from new contribute link

Docusaurus already renders an icon

pwizla committed:
Bump version to 4.23.1

b3nks committed:
Update 11ty.md (#2088)

With eleventyComputed the Data Value for Title and Name can be used in Frontmatter.

pwizla committed:
Add missing available options for the client database configuration (#2084)

* Add missing available options for the client database configuration

* Update docusaurus/docs/dev-docs/configurations/database.md

Co-authored-by: Ben Irvin <ben@innerdvations.com>

---------

Co-authored-by: Ben Irvin <ben@innerdvations.com>

MbonuJennifer committed:
updated icons to include links as well on homepage (#2090)

nc1z committed:
Document using strapi instance for migrations (#2085)

* docs: using strapi instance for migrations

* docs: update database-migrations.md

might result in data loss so the highest-level of callout is required

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>

* docs: title clean up for database-migrations.md

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>

---------

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>

imyuanx committed:
fix: fix example code (#2094)

pwizla committed:
Add temp. redir. while corporate website is hot fixed

pwizla committed:
Fix JSON from latest redir. commit

pwizla committed:
Add an intro. to sanitization and validation

pwizla committed:
Add temp. redir. for SSO link from corp. website

pwizla committed:
Update single-sign-on.md

pwizla committed:
Update review-workflows.md

pwizla committed:
Clean up temporary redirs

pwizla committed:
Re-add temp. redir.

pwizla committed:
Re-add temp. redir. while corp. website is re-deployed

lcpichette committed:
pr: chore - Improving routes page (#2097)

* Update routes.md

* Fixes typo in command

* Updates other controllerName.actionName -> api::apiName.controllerName.functionName

* Update docusaurus/docs/dev-docs/backend-customization/routes.md

* Update docusaurus/docs/dev-docs/backend-customization/routes.md

* Update docusaurus/docs/dev-docs/backend-customization/routes.md

* Update docusaurus/docs/dev-docs/backend-customization/routes.md

* Update docusaurus/docs/dev-docs/backend-customization/routes.md

* Update docusaurus/docs/dev-docs/backend-customization/routes.md

* Update docusaurus/docs/dev-docs/backend-customization/routes.md

* Update docusaurus/docs/dev-docs/backend-customization/routes.md

* Update docusaurus/docs/dev-docs/backend-customization/routes.md

* Update docusaurus/docs/dev-docs/backend-customization/routes.md

* Update docusaurus/docs/dev-docs/backend-customization/routes.md

* Update docusaurus/docs/dev-docs/backend-customization/routes.md

* Update docusaurus/docs/dev-docs/backend-customization/routes.md

---------

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>

MbonuJennifer committed:
Estimation of time to complete all the steps at the beginning of the Quick Start Guide (#2098)

* Time estimate task for quick sart guide

* Re-edited text, now we have just one time estimete text

derrickmehaffy committed:
Security information update to users-permissions

derrickmehaffy committed:
add images

derrickmehaffy committed:
Merge pull request #2107 from strapi/sec/up

MbonuJennifer committed:
Rewording of TOC/sidebar introduction labels (#2111)

* Rewording of TOC/sidebar introduction labels

* Update docusaurus/sidebars.js

* Update docusaurus/sidebars.js

* Update docusaurus/sidebars.js

* Update docusaurus/sidebars.js

* Update docusaurus/sidebars.js

---------

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>

projectashik committed:
Update admin-panel-customization.md (#2105)

Replaced "," with "." in file name in vite config for Typescript.

qraxiss committed:
fix: import should be same directory ../ -> ./ (#2104)

voicewise committed:
Update database.md (#2108)

The configuration parameters required by mysql2 are "name" and "password". The parameter "username" rises a warning.

benhovinga committed:
Fixed file structure alignment (#2102)

The components folder was pushed too far to the right because of duplicate hierarchy columns.

MbonuJennifer committed:
Created component for numbered lists referring to annotated screenshots (#2099)

* Created numbered images reference

* Use case of Screenshot reference

* Use case of Screenshot reference

* Use case of Screenshot reference

* Use case of Screenshot reference

pwizla committed:
Fix middlewares docs

pwizla committed:
Fix missing space

pwizla committed:
Mention findMany returns object for single types in ES API

Commit:
Duplicate deploy on push option for Update repository modal documentation

kynatro committed:
Update custom fields options description type (#2115)

Update the type for the description field for custom field option items to reflect actual implementation.

pwizla committed:
Update package.json

cache-your-dreams committed:
Merge pull request #2114 from strapi/cloud/update-repo-modal

Addition of "deploy on push" option in Update Repository documentation

Juuhz committed:
Update admin-redirect.md (#2113)

Adjust path

pwizla committed:
Fix broken links

pwizla committed:
Fix typo

pwizla committed:
Update package.json

pwizla committed:
Improve limitations note in custom fields docs

pwizla committed:
Update CONTRIBUTING.md

pwizla committed:
Swap NPM & yarn tabs order in the QSG

pwizla committed:
Update intro.md

pwizla committed:
PostHog integration (#2122)

* Add Posthog plugin

* Add placeholder config. for Posthog

* Update

* Update docusaurus/docusaurus.config.js

pwizla committed:
Update docusaurus.config.js

pwizla committed:
Update docusaurus.config.js

Commit:
Cloud CLI documentation (#2086)

* Write 1st draft of Cloud CLI + Update TOC

* Improve and complete Cloud CLI documentation

* Create file + Write guide for deploying projects with CLI

* Update deploy introduction + project deploy guide for Cloud dashboard

* Update TOC

* Update TOC + Comment some paragraphs that may not be included in final version

* Improve wording

* Update wording for "hosted on remote git repo"

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>

* Improve wording again for "hosted on remote git repo"

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>

* Improve wording again

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>

* Update docusaurus/docs/cloud/cli/cloud-cli.md

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>

* Keep improving wording

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>

* Remove unnecessary comments

* Update docs after blitz session test

* Improve after review

* Add link to Dev Docs + Fix wording

* Quick Start Guide and CLI installation guide updates for the Cloud CLI (#2119)

* Update the Quick Start Guide to reflect new Cloud CLI behavior

* Update the CLI installation docs

* Add cross-link to QSG to CLI install. guide

* Update docusaurus/docs/dev-docs/quick-start.md

* Update docusaurus/docs/dev-docs/quick-start.md

* Update docusaurus/docs/dev-docs/quick-start.md

* Update docusaurus/docs/dev-docs/quick-start.md

* Update docusaurus/docs/dev-docs/installation/cli.md

Co-authored-by: Ben Irvin <ben@innerdvations.com>

* Update docusaurus/docs/dev-docs/installation/cli.md

Co-authored-by: Ben Irvin <ben@innerdvations.com>

* Update docusaurus/docs/dev-docs/quick-start.md

Co-authored-by: Ben Irvin <ben@innerdvations.com>

* Update docusaurus/docs/dev-docs/quick-start.md

Co-authored-by: Ben Irvin <ben@innerdvations.com>

* Update docusaurus/docs/dev-docs/installation/cli.md

* Update docusaurus/docs/dev-docs/installation/cli.md

* Update docusaurus/docs/dev-docs/installation/cli.md

* Update docusaurus/docs/dev-docs/installation/cli.md

* Update docusaurus/docs/dev-docs/installation/cli.md

* Update docusaurus/docs/dev-docs/installation/cli.md

* Update docusaurus/docs/dev-docs/installation/cli.md

* Update docusaurus/docs/dev-docs/quick-start.md

* Swap NPM & Yarn tabs order

* Shorten text

---------

Co-authored-by: Ben Irvin <ben@innerdvations.com>

* Add "Updated" badge to QSG in TOC

* Update completion time in QSG

---------

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>
Co-authored-by: Ben Irvin <ben@innerdvations.com>
Co-authored-by: Pierre Wizla <pwizla+github@gmail.com>

pwizla committed:
Bump version number to 4.25.0

pwizla committed:
Mention you can't upload HTML files during trial

pwizla committed:
Update deployment.md

pwizla committed:
Update error-handling.md

pwizla committed:
Amend database docs about external databases

pwizla committed:
Update database.md

pwizla committed:
Mention Cloud CLI commands in Dev Docs CLI reference

pwizla committed:
Fix preposition usage in prev. commit

abuhelos committed:
Digital Ocean Droplet Deployment Edit: Add info about nginx max file size to config instructions (#2132)

* Add info about nginx max file size to config instructions

* Update docusaurus/docs/dev-docs/deployment/digitalocean.md

---------

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>

pwizla committed:
Edit top banner for Strapi 5 RC

abdallahmz committed:
Update the cloud cli logout step (#2134)

Related to https://github.com/strapi/strapi/pull/20502

Commit:
Billing flow revamp (#2130)

* Add new icons in project settings page

* Add icon for Variables

* Update documentation for Profile > Billing

* Add documentation for Settings > Plans

* Add new icons in Profile settings page + some improvements

* Update/add documentation about Invoices

* Add comments to update screenshots at the end of the project

* Change Usage & Billing name everywhere + start documenting seats management

* Document seats management

* Create + Use snippet for invoices statuses info

* Update TOC badges

* Fix typos and errors after reviews

* Update & add screenshots

---------

Co-authored-by: Pierre Wizla <pwizla@users.noreply.github.com>

pwizla committed:
Update to Cloud CLI docs re. connecting a git repo. (#2128)

* Update to Cloud CLI docs re. connecting a git repo.

* Add mention for all deploy. methods

* Fix heading link

* Update deploys intro.

pwizla committed:
Merge branch 'repo/five-rc'

pwizla committed:
Capitalize RC (#2136)

pwizla committed:
Add PostHog React test

pwizla committed:
Add fetch-related blog post link to REST API guides list (#2139)

pwizla committed:
Add link to blog post about using fetch() with the REST API

pwizla committed:
Add redir. for REST API guides intro.

pwizla committed:
Add FAQ entry about Strapi Cloud upgrade (#2140)

pwizla committed:
Update usage-billing.md

pwizla committed:
Update usage-information.md

pwizla committed:
Remove duplicate TOC items

pwizla committed:
Update QSG to match new CLI behavior (#2133)

pwizla committed:
Bump version number to 4.5.2

gonbaum committed:
feat(cli): strapi projects

gonbaum committed:
Merge pull request #2144 from strapi/cli/projects-list

feat(cli): strapi projects

pwizla committed:
Update Algolia API key

pwizla committed:
Merge branch 'main' of github.com:strapi/documentation

zlaazlaa committed:
Merge branch 'strapi:main' into main

pwizla commented:
Hi 👋  It's been a while 😅 I'm closing this PR as it points to a very old version of the documentation, and it was fixed since in another PR. Thank you again very much for your valuable contribution 🙏
